pkg/memtier/prompt.go

```diff
@@ -698,10 +698,10 @@ PGTABLE       %d
 			})
 		return csOk
 	}
-	p.aranges = ar
-	p.output("aranges = <%d address ranges of pid %d>\n", len(ar.Ranges()), ar.Pid())
+	p.adranges = ar
+	p.output("adranges = <%d address ranges of pid %d>\n", len(ar.Ranges()), ar.Pid())
```

Comment:
Even more important than keeping the internal variable name connected to the user command, it is to keep user language consistent. "arange" is the command.

But as said, let's keep also the internal variable named "aranges", too.